extension/.vscode/settings.json

```diff
@@ -1,3 +0,0 @@
-{
-    "typescript.tsdk": "node_modules\\typescript\\lib"
-}
```

Comment:
I don't think you want to remove this...

Reply:
ah commited that by mistake

Reply:
actually that file should no longer be needed

Reply:
Why not?

Reply:
Hmm, your assumption is that VSCode will "do the right thing" in that case?  Happen to have any docs that support that, because I feel like in my experience VSCode still used its built-in even when I had a `node_modules` in the root.

Reply:
well I don't think that file did anything as it was alone in `extension/` folder. Not sure why vscode would look it from there?

Reply:
Oh, I see the problem now!  Sorry, I thought this was `<root>/.vscode/settings.json`